src/V3Options.cpp

```diff
@@ -1500,6 +1500,7 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc,
     DECL_OPTION("-v", CbVal, [this, &optdir](const char* valp) {
         V3Options::addLibraryFile(parseFileArg(optdir, valp));
     });
+    DECL_OPTION("-valgrind", CbCall, []() {});  // Processed only in bin/verilator shell
```

Comment:
We'll need a test to cover this code - maybe just call bin/verilator --help --valgrind just to make sure it's ignored.